components/TechnologyTag/index.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  EnergyStorageIcon,
+  GeothermalIcon,
+  HydroelectricIcon,
+  LandBasedWindIcon,
+  OffshoreWindIcon,
+  PumpedStorageIcon,
+  SolarPvIcon,
+} from '../../assets/Technology-Tag-Icons/icons';
+import { TagText1 } from '../../styles/texts';
+import { TechnologyTagStyles } from './styles';
+
+export default function TechnologyTag({
+  technology,
+}: {
+  technology: string | undefined;
+}) {
+  const iconMap: { [key: string]: JSX.Element } = {
+    'Solar PV': <SolarPvIcon />,
```

Comment:
For each SVG icon component, you will now need to pass in the fill and stroke props. Below is an example using Offshore Wind (but just make sure you use the colors.ts global colors stylesheet. it should look something like fill={COLORS.navy})

<img width="565" alt="Screenshot 2024-11-11 at 11 03 35 AM" src="https://github.com/user-attachments/assets/2cd7a598-2688-4401-a7d0-eb2ce0a7efac">